code/Cargo.toml

```diff
@@ -380,7 +380,7 @@ cosmwasm-std = { git = "https://github.com/dzmitry-lahoda-forks/cosmwasm", rev =
 ] }
 cw2 = { git = "https://github.com/dzmitry-lahoda-forks/cw-plus", rev = "458e2eb014253d2131219e518c64475a8348c5a3", default-features = false }
 
-serde = { version = '1.0.136', default-features = false, features = ["derive"] }
+serde = { version = '1.0.181', default-features = false, features = ["derive", "unstable"] }
```

Comment:
why is this unstable?

Reply:
Because `error_in_core` is a nightly Rust feature.  With neither `std` nor `unstable` feature, serde defines its own StdError trait which is different from std::error::Error and this doesn’t work well with the rest of the CosmWasm which assumes std::error::Error is used.